src/components/alert/alert-component.ts

```diff
@@ -128,6 +140,29 @@ export class AlertCmp {
     }
   }
 
+  ionViewPreLoad() {
+    if (this.d.subview && this.d.subview.component) {
+
+      const componentFactory = this._cfr.resolveComponentFactory(this.d.subview.component);
+
+      const componentProviders = ReflectiveInjector.resolve([
+        { provide: NavParams, useValue: new NavParams(this.d.subview.args) }
+      ]);
+      const childInjector = ReflectiveInjector.fromResolvedProviders(componentProviders, this._viewport.parentInjector);
+
+      // ******** DOM WRITE ****************
+      const componentRef = this._viewport.createComponent(componentFactory, this._viewport.length, childInjector, []);
+      this.instance = componentRef.instance;
```

Comment:
since the instance reference is already managed by ViewController, let's change `this.instance` to `this.isCustomAlert :  boolean` (by default it is `false`)
```
this.isCustomAlert = true;
```